README.md

```diff
@@ -32,4 +35,11 @@ curl http://localhost:8081/info
 
 ## Database
 
-The database will automatically be created when starting the application.
\ No newline at end of file
+The database will automatically be created when starting the application.
+
+## Testing
+Running the tests requires a running instance of the database. The easiest way to do this is run the application up 
```

Comment:
I wonder if there is a way for tox to do this?

Reply:
Possibly could do, I'll have a look but initially it was easiest to split out the docker-compose into travis and then call the tests from tox. I'll look into this in another pull request.